src/config/styles/html.js

```diff
@@ -6,22 +6,26 @@ import { imageHeight } from '../../helpers/imageHelper';
 
 export const html = {
   p: {
+    margin: 0,
```

Comment:
can we see how this results?

![image](https://user-images.githubusercontent.com/1942953/222184682-a6f71308-262b-49f8-84e5-6c4dbc713e93.png)

the data used is without html notations
![image](https://user-images.githubusercontent.com/1942953/222184757-03c40a38-47f1-499a-a351-6e608b5635e0.png)

so there is a reason, that we do not have an empty line anymore. but for some reason, this was the case before.

if there are other informations in other entries with more empty lines, that will be removed after merge, this could be bad. any ideas how to avoid that?

Reply:
i remember that there are configs on how to handle line breaks? is that true? or maybe there was such config in the old version and handled differently now? are there issues according that?

other idea is to check, if the description contains html, than render html and otherwise render text. maybe this remains the line breaks?

Reply:
i will look at the document again in detail. i did not see a configuration as you say.

Reply:
I added `HTML_REGEX` and if we have a content without html tag, we show `RegularText`. Do you think this is appropriate?
|before|after|
|--|--|
![IMG_2641](https://user-images.githubusercontent.com/11755668/222713297-67c35dca-f2ac-4c9e-aef7-bc2abd45af3d.jpg) | ![IMG_2640](https://user-images.githubusercontent.com/11755668/222713316-c456a867-8add-46ca-887f-a8884ba7c4fb.jpg)


https://github.com/smart-village-solutions/smart-village-app-app/commit/fed00e32327acd9474e3c571abf33a7a4fee0c04

Reply:
yes! this seems pretty good!